frontend/header.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/scss/style.css">
+    <title></title>
+</head>
+<body>
+    <header class="header">
+          <nav class="header__nav">
+          <img class="header__nav-logo" src="/design/Logos/Logo_wb.png" alt="logo" height="53.02"></a>
```

Comment:
the height here seems strange

Reply:
I took the size of the logo from Figma prototype

Reply:
yeah but px is missing :)